test-e2e/web/README.md

```diff
@@ -0,0 +1,55 @@
+# Simple Web App with Wasm
+
+## Preparations
+
+Follow these steps only if there are changes in the `mopro-wasm` package. Otherwise, these steps are not required.
+
+### Compile Halo2 Circuits to Wasm
+
+This web test page requires three Wasm circuits: `halo2-plonk-fibonacci`, `halo2-hyperplonk-fibonacci`, and `halo2-gemini-fibonacci`.  
+Refer to [`mopro-wasm/README.md`](../../mopro-wasm/README.md) for more details.
+
+Use `wasm-pack` to generate halo2 wasm package for this testing web app with the following commands:
+
+```bash
+mopro-wasm $ wasm-pack build --target web --out-dir ../test-e2e/web/mopro-pkg  -- --all-features
+```
+
+### Copy Parameters
```

Comment:
I tried using relative paths, but the server cannot serve files located outside the parent directory. It probably worked with symbolic link with `--symlink` flag in `serve`. I will update it if it will works.

Reply:
Fixed at [514bd4d](https://github.com/zkmopro/mopro/pull/254/commits/514bd4db15d2d37b70015bfc25558388f67e3d99) without using any **symbolic links**.
will update docs in the following commits.